MetasysServices.Tests/MetasysClientTests.cs

```diff
@@ -282,26 +285,10 @@ public void TestGetObjectIdentifierBadRequestThrowsException()
             PrintMessage($"TestGetObjectIdentifierBadRequestThrowsException: {e.Message}");
         }
 
-        [Test]
```

Comment:
This test was removed because it represents a condition that can't happen any more. We can't fail parsing a guid since we no longer attempt to parse to guid.